lib/vsc/install/shared_setup.py

```diff
@@ -406,7 +408,7 @@ def get_name_url(self, filename=None, version=None, license_name=None):
             self.private_repo = True
 
         if 'url' not in res:
-            allowed_remotes = ', '.join(GIT_REMOTES)
+            allowed_remotes = ', '.join(['{}/{}'.format(*remote) for remote in GIT_REMOTES])
```

Comment:
eugh `'/'.join(remote)` of zelfs `'%s/%s' % remote`?

Reply:
done in https://github.com/hpcugent/vsc-install/pull/190/commits/2ae7fb2ce7a9e0e9b575526f2047bfdd56e9dcc7